src/components/UserActions/QueryList.ts

```diff
@@ -9,14 +9,15 @@ const DEFAULT_TRANSFORMATION = () => (query: string) => {
     container.classList.add('coveo-list-row');
 
     const icon = document.createElement('div');
+    icon.classList.add('coveo-row-icon');
     icon.innerHTML = search;
     
-    const span = document.createElement('span');
-    span.classList.add('coveo-content');
-    span.innerHTML = query;
+    const link = document.createElement('a');
```

Comment:
Why do we change the span node to a link node ?

Reply:
Neil requested the change so we could more easily highlight the fact that it was clickable. 
Link to discussion: https://coveord.atlassian.net/browse/SFINT-2368

Reply:
👍